addons/l10n_do/data/account_data.xml

```diff
@@ -9,41 +9,14 @@
         <record id="group_isr" model="account.tax.group">
             <field name="name">ISR</field>
         </record>
-        <record id="group_ret" model="account.tax.group">
-            <field name="name">Retenciones</field>
-        </record>
         <record id="group_tax" model="account.tax.group">
             <field name="name">Otros Impuestos</field>
         </record>
         <record id="tax_group_isc" model="account.tax.group">
             <field name="name">ISC</field>
         </record>
-        <record id="tax_group_itbis_0" model="account.tax.group">
```

Comment:
the tax group is still referenced in two taxes

Reply:
After this PR? https://github.com/odoo/odoo/pull/27172/files

Reply:
No indeed. Will wait for #27172 to be integrated first.